src/Components/PaperEditor/ProgrammeScriptContainer/index.js

```diff
@@ -215,7 +230,23 @@ const ProgrammeScriptContainer = (props) => {
     return elements.indexOf(insertElement);
   };
 
-  // // [old comment] TODO: needs to handle when selection spans across multiple paragraphs
```

Comment:
Maybe we shouldn't get rid of this comment?

Reply:
I'll open a new ticket - probs best to have TODOs on the board v in the comments